lib/sorting.js

```diff
@@ -65,7 +66,21 @@ function sortDeclarations(a, b) {
 	return a.initialIndex - b.initialIndex;
 }
 
+function isShorthand(a, b) {
+	const longhands = shorthandData[a] || [];
+
+	return longhands.indexOf(b) >= 0;
```

Comment:
A little bit better readability:

```suggestion
	return longhands.includes(b);
```

Reply:
I'm for it, but it requires a pretty recent version of node. OK with that?

Reply:
It [supports](https://node.green/#ES2016-features-Array-prototype-includes-Array-prototype-includes) since Node.js 6.4.0. This plugin supports Node.js 8+ :)